modules/prebidServerBidAdapter/index.js

```diff
@@ -641,6 +641,9 @@ const OPEN_RTB_PROTOCOL = {
         }
         utils.deepSetValue(request, 'regs.ext.gdpr', gdprApplies);
         utils.deepSetValue(request, 'user.ext.consent', firstBidRequest.gdprConsent.consentString);
+        if (firstBidRequest.gdprConsent.addtlConsent && typeof firstBidRequest.gdprConsent.addtlConsent === 'string') {
+          utils.deepSetValue(request, 'user.ext.ConsentedProvidersSettings.consented_providers', firstBidRequest.gdprConsent.addtlConsent);
```

Comment:
Just to confirm - are we sure that this field is supported by PBS?

Reply:
that was the field @bretg proposed on https://github.com/prebid/Prebid.js/issues/5389#issuecomment-669344015 bullet point 2